portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -13,6 +13,10 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.sps.data.Task;
```

Comment:
You don't use this

Reply:
I think I need them in order to use Entity commentEntity = ... 
Otherwise it will not compile, it appears as "cannot find symbol"

Reply:
Was that error from a previous version of the code? You don't use Task in this file at all, so it can't complain about finding a symbol that doesn't exist.

Reply:
Oh okay, I thought you were referring to the 4 lines, but I now get what you meant, thank you Yi-Tao